.github/workflows/build-client-packages.yml

```diff
@@ -0,0 +1,126 @@
+name: Build client packages
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - 'v*'
+  pull_request:
+    branches:
+      - master
+
+jobs:
+
+  build-packages:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Get the version number for the packages
+        id: get_version
+        # If this is a tag, use the tag name (e.g. v1.2.3) without v as version number.
+        # Otherwise, just use 0.0.
```

Comment:
There's no file that contains the version in the packages themselves?

Reply:
Nope, the argument passed to the package build action should be the only place where we need the version number.

Reply:
I guess it could, but why would you want that?

Reply:
It's basically managed by the package manager of the OS then.
This is an example of the "official" cvmfs-config rpm for the default cern repos:
https://ecsft.cern.ch/dist/cvmfs/cvmfs-config/cvmfs-config-default-2.0-1.noarch.rpm

Reply:
It could make it clear which version of the configuration is installed easily?
Maybe not that useful, since you can just check which version of the package is installed, I guess...

Reply:
I'd say that the package version indeed provides this information.